src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -160,29 +283,55 @@ export class AllRecords extends React.Component<IAllRecordsProps, IAllRecordsSta
               order={this.state.order}
               sortFunc={this.sort}
             />
-            {/* d-none until implemented */}
-            <div className="pill d-none">
-              <span>
-                <FontAwesomeIcon icon="th" />
-                {' | '}
-                <FontAwesomeIcon icon="bars" />
-              </span>
-            </div>
             <ButtonPill onClick={this.toggleFilter} translate="providerSite.filter" />
           </div>
         </div>
-        <Row noGutters>
-          <MediaQuery maxDeviceWidth={768}>{this.mapRecords(allRecords)}</MediaQuery>
-          <MediaQuery minDeviceWidth={769}>{filterOpened ? this.mapWithFilter(allRecords) : this.mapRecords(allRecords)}</MediaQuery>
-        </Row>
-        <div className="button-pill mb-4 text-center">
-          <div
-            className={`d-inline button-pill ${hasReachedMaxItems ? 'disabled' : ''}`}
-            onClick={() => this.handleLoadMore(hasReachedMaxItems)}
-          >
-            <Translate contentKey="providerSite.loadMore" />
+        {isMapView ? (
+          <Row className="mb-4">
+            <Col md={isRecordHighlighted || filterOpened ? 8 : 12} className="pb-2">
+              <Map
+                googleMapURL={mapUrl}
+                records={allRecords}
+                lat={selectedLat}
+                lng={selectedLng}
+                loadingElement={<div style={{ height: `100%` }} />}
+                containerElement={<div style={{ height: `400px` }} />}
```

Comment:
Are you sure that fixed value works with responsive view?

Reply:
yes